insights/client/connection.py

```diff
@@ -459,7 +459,8 @@ def branch_info(self):
         response = self.session.get(self.branch_info_url)
         logger.debug("GET branch_info status: %s", response.status_code)
         if response.status_code != 200:
-            raise LookupError("Bad status from server: %s" % response.status_code)
+            logger.error("Bad status from server: %s" % response.status_code)
```

Comment:
Loggers use varargs to defer interpolation of values so please do the following:

`logger.error("Bad status from server: %s", response.status_code)`